apps/src/code-studio/components/progress/StageProgress.jsx

```diff
@@ -61,6 +62,10 @@ class StageProgress extends Component {
     const {stageExtrasUrl, onStageExtras, stageTrophyEnabled} = this.props;
     let levels = this.props.levels;
 
+    const inMiniRubricExperiment = experiments.isEnabled(
```

Comment:
i think this looks good as-is, but if performance is still an issue, i think we can remove checking the experiment in the `render()` method and instead set some state in `componentDidMount` like this:

```
componentDidMount() {
  this.state = {inMiniRubricExperiment: experiments.isEnabled(experiments.MINI_RUBRIC_2019)};
}

render() {
  const inMiniRubricExperiment = this.state.inMiniRubricExperiment;
}
```

there are a few places below where we could make this change as well.